src/pages/workspace/reportFields/WorkspaceReportFieldsPage.tsx

```diff
@@ -66,6 +67,14 @@ function WorkspaceReportFieldsPage({
     const [selectedReportFields, setSelectedReportFields] = useState<PolicyReportField[]>([]);
     const [deleteReportFieldsConfirmModalVisible, setDeleteReportFieldsConfirmModalVisible] = useState(false);
 
+    const fetchReportFields = useCallback(() => {
```

Comment:
Do we need `useCallback()`? React compiler will memoize the function for us, right?

Reply:
It should, but as its an experimental thing additional performance security should be fine here